docs/app/client/carousel/props.json

```diff
@@ -57,6 +57,11 @@
         "name": "isRtl",
         "type": ["boolean"],
         "description": "Render Props: Set right to left (rtl) view"
+      },
+      {
+        "name": "isSwipeDragDisabled",
```

Comment:
Potential breaking change. Mentioned in AutoSlide example

Reply:
Removed.